### ISSUE TITLE
Add optional Mathjax to Tusky

### ISSUE THREAD

**Issue by abdullahkhalids** [tuskyapp/Tusky#3122]
Latex style mathematics on the web requires loading Mathjax or Katex javascript libraries. An example of this is this [toot](https://mathstodon.xyz/@littmath/109592640136010318), which looks like this in a web browser

![image](https://user-images.githubusercontent.com/2965278/210017350-e141cf01-351c-4d4a-8654-c6d46acecda9.png)

In Tusky, these libraries are not loaded, so the latex syntax is shown as is. This is what the toot looks like in Tusky

![image](https://user-images.githubusercontent.com/2965278/210017304-9b716810-7226-48ae-89cd-1084f0eb108b.png)

Please add an option to loads mathjax or katex libraries inside Tusky. 

* * * *
- Tusky Version: 20.0
- Android Version: Any
- Android Device: Any
- Mastodon instance (if applicable): N/A

- [ x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by mcclure** [comment]
I would really like this actually

**Comment by connyduck** [comment]
Yes, this would be nice to have. 
1) We need Android Java or Kotlin libraries for this though, can't have JavaScript in Tusky
2) It would be nice if there where some kind of flag in the Api that lets us know that a post contains LaTeX so we render it as the user intended

**Comment by mcclure** [comment]
There seem to be a couple existing Android libraries

@reedbeta@mastodon.gamedev.place sent me this helpful link that might help us understand exactly what subset is in use on Mathstodon "links at the bottom to lists of supported TeX commands and stuff" https://docs.mathjax.org/en/latest/input/tex/index.html#tex-support

**Comment by abdullahkhalids** [comment]
>     1. We need Android Java or Kotlin libraries for this though, can't have JavaScript in Tusky

Hmmm. This certainly complicates things. 

Usually Mathjax converts the latex mathematics into styled html that is rendered by the browser as usual. This has the advantage that zooming keeps the relative size of the text and math constant.

There are [android-friendly](https://github.com/NanoMichael/MicroTeX)  libraries for latex, but they convert each latex expression to its own svg image.

**Comment by kilipan** [comment]
How about this one?
https://github.com/tech-pw/RenderX

(I'm not a software dev so no idea how suitable it is, just came across this issue while looking for a way to display latex in tusky.)
